enterprise_access/apps/api_client/lms_client.py

```diff
@@ -105,6 +105,44 @@ def get_enterprise_customer_data(self, enterprise_customer_uuid=None, enterprise
             logger.exception(exc)
             raise
 
+    def create_enterprise_customer(self, *, name, slug, country, **kwargs):
+        """
+        Creates a new enterprise customer record.
+
+        Arguments:
+            name (string): The name of the customer.
+            slug (string): Slug of the enterprise customer.
+            country (string): The country code of the customer.
+            kwargs (dict): Any other fields to specify for the newly-created customer.
+        Returns:
+            dictionary containing enterprise customer metadata
+        """
+        payload = {
+            'name': name,
+            'slug': slug,
+            'country': country,
+            'site': {
+                'domain': settings.DEFAULT_CUSTOMER_SITE,
+            },
+            **kwargs,
+        }
+        try:
+            response = self.client.post(
+                self.enterprise_customer_endpoint,
+                json=payload,
+                timeout=settings.LMS_CLIENT_TIMEOUT,
+            )
+            response.raise_for_status()
+            payload = response.json()
+            logger.info(
+                'Successfully created customer %s with data %s',
+                name, payload,
```

Comment:
nit: similar to Troy's comment above, should this log include the customer UUID (given name is not unique)?

Reply:
The uuid should be in `payload`.